CHANGELOG.md

```diff
@@ -1,6 +1,58 @@
 # Changelog
 All notable changes to this project will be documented in this file. This CHANGELOG roughly follows the guidelines from [www.keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [v0.10.2]
+### Added
+- Approximate font metrics only when metrics don't exist (#1898)
+- Add KaTeX version to stylesheet and troubleshooting guide (#1893)
+- Add symbol double square brackets (#1947)
+- Support double-square curly braces (#1953)
+
+ ### Changed
+- Upgrade minimum development Node version to v8 (#1861)
+- Disable @babel/env debug (#1874)
+- Add issue templates (#1862)
+- Added 'katex-element' (#1905)
+- Fix Users' logo and url (#1896)
+- Load fonts before running screenshotter (#1891)
+- Add Browserstack logo (#1879)
+- Added Android library (#1943)
+- Move custom colors used by Khan into macros.js (#1933)
+- Remove redundant symbols from #1947 (#1954)
```

Comment:
Not important, but we could just list this under the `Add symbol double square brackets` bullet like `(#1947, #1954)`.

Reply:
Will do.